test/plugin/scenarios/canal-scenario/bin/startup.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+home="$(cd "$(dirname $0)"; pwd)"
+
+java -Dcanal.host=${CANAL_HOST} -Dcanal.port=${CANAL_PORT} -jar ${agent_opts} "-Dskywalking.agent.service_name=canal-scenario" ${home}/../libs/canal-scenario.jar &
```

Comment:
We don't need to override configuration of agent  next time when the `server_name` is the same as `scenario_name`.

Reply:
Let's keep this in the doc as recommendation.

Reply:
> We don't need to override configuration of agent next time when the `server_name` is the same as `scenario_name`.

deleted